web/src/portico/landing-page.js

```diff
@@ -194,7 +195,9 @@ $(() => {
     events();
 
     if (window.location.pathname === "/team/") {
-        render_tabs();
+        const contributors = page_params.contributors;
+        delete page_params.contributors;
+        render_tabs(contributors);
```

Comment:
This commit is broken, because the `render_tabs` function has not been modified to accept this parameter.